src/pages/steps/Steps.tsx

```diff
@@ -195,7 +200,7 @@ const Steps = ({
   };
 
   return (
-    <Stack spacing={5} width="100%">
+    <Stack {...getStackProps(props)} spacing={5} width="100%">
```

Comment:
Its more useful to spread the `getStackProps` after existing props, so they can be overridden if needed